docs/modules/ROOT/pages/documentation/payara-server/system-properties.adoc

```diff
@@ -16,11 +16,13 @@ libraries from applications, and
 _latexmath:[${Domain}_/lib will override the library in _$]\{Product-Root}_/modules
 directory | true, false | true
 
-| fish.payara.shutdowngrace | Integer | After server shutdown is initiated the server waits this period of time to finish ongoing requests before terminating. Currently, new requests are still accepted during this grace period.| 0 - Integer.MAX | 0
+| fish.payara.admin.command.jline.log.disable | Boolean | When set to true, JLine logging entries for invalid key bindings and other unwanted errors will be prevented - This is a property for the asadmin tool | true, false | false
+
+| fish.payara.deployment.transform.namespace | Boolean | When set to true, Namespace transformation (jakarta.* to javax.*) is always performed, If value is undefined then auto transformation is performed on the detection of jakarta.* classes in the deployment archive otherwise transformation task is disabled. | true, false, null | null (undefined)
```

Comment:
The `*` character must be either escaped or be in a code snippet, otherwise it turns on **bold** style.

```suggestion
| fish.payara.deployment.transform.namespace | Boolean | When set to true, Namespace transformation (`jakarta.*` to `javax.*`) is always performed. If value is undefined then auto transformation is performed on the detection of `jakarta.*` classes in the deployment archive otherwise transformation task is disabled. | true, false, null | null (undefined)
```